### DISCUSSION TITLE
Authentik in rootless Podman?

### DISCUSSION THREAD

**Discussion by Pshemas** [goauthentik/authentik#9759] [General]
Has anyone here got Authentik working with Podman (podman compose)?

I've been looking at compose.yaml and this caught my eye:

```
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock
```

As I don't have docker installed I don't have docker.socket. I could map /var/run/podman/podman.socket... but I guess that won't work in rootless mode as this file belongs to root/root.

Any suggestions?

**Comment by gwelch-contegix** [comment]
You don't need the socket, it's for automatically creating containers. you instead create all the containers manually. You should also be able to start the user service https://github.com/containers/podman/blob/main/docs/tutorials/socket_activation.md and it might be compatible, I know it's compatible with the docker-compose command.

**Comment by rtk0c** [comment]
Late addition: in addition to the rootful `/run/podman/podman.sock` activated with `systemctl enable --now podman.sock` (as you seem to already know) a there is also a per-user sock, at `/run/user/$UID/podman/podman.sock` activated with <code>systemctl <b>--user</b> enable --now podman.sock</code>. You should be able to bind mount that sock to `/var/run/docker.sock` as Authentik expects, though I haven't tested if what's spoken across the socket is compatible.

Hope this is helpful.

**Comment by maurerle** [comment]
It works using `/run/user/$UID/podman/podman.sock` - make sure to resolve this to `/run/user/1001/podman/podman.sock` in the compose.yml - otherwise it tries to resolve this internally (as `/run/user/podman/podman.sock`), which is wrong.

Authentik works fine with podman :)
